GUI/Controls/ModInfoTabs/Metadata.cs

```diff
@@ -72,10 +72,10 @@ public void UpdateModInfo(GUIMod gui_module)
                     ReplacementTextBox.Text = module.replaced_by.ToString();
                 }
 
-                if (gui_module.DownloadCount == null)
+                if (gui_module.DownloadCount == null or gui_module.DownloadCount == 0)
```

Comment:
`or` isn't valid C# syntax, you need ` || ` instead.
Are you using an IDE like Visual Studio or JetBrains Rider? it'd show these kinds of issues by underlining them in red before you post them.

Reply:
If you want to use `or`, you can do this:

```suggestion
                if (gui_module.DownloadCount is null or 0)
```

Reply:
> If you want to use `or`, you can do this:
What's this fancy new syntax lol, you learn something new every day.

Reply:
> What's this fancy new syntax lol, you learn something new every day.

Patterns! They can be confusing at first, but I've grown to love them:

- <https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/patterns>